examples/laser_ionization/analysis_laser_ionization.py

```diff
@@ -86,20 +86,60 @@
 Tz2_c = all_data_circular['[uz^2]'][0,0]*scc.m_e*scc.c**2/scc.e
 temp_eV_circular = 1./3*(Tx2_c+Ty2_c+Tz2_c)
 
+# calculation of temperature with Open-PMD viewer diagnostics
+wf_l, _ = ts_linear.get_field(field='w_elec', iteration=iteration)
+
+ux2_linear, _ = ts_linear.get_field(field='ux^2_elec', iteration=iteration)
+ux2_mean_linear = np.sum(np.sum(ux2_linear*wf_l, axis=1),axis=1) / np.sum(np.sum(wf_l, axis=1),axis=1)
+ux2_average_linear = statistics.mean(ux2_mean_linear[0:10])
```

Comment:
I think this code could be simplified like this. Not tested, please check it is correct. This should also be applied below consistently.
```suggestion
izmax = 10
wf, _ = ts_linear.get_field(field='w_elec', iteration=iteration)
sum_wf = np.sum(wf, axis=(1,2))[0:izmax]

ux2, _ = ts_linear.get_field(field='ux^2_elec', iteration=iteration)
ux2_average_linear = np.average(np.sum(ux2, axis=(1,2))[0:izmax], weights=sum_wf)
```
- Let's not import a new package `statistics` if all we need is in `numpy`.
- The same polarization is sometime labeled as `_l` and sometimes as `_linear`. For temporary variables, we probably don't need to label it, so I removed some occurrences.
- Further simplification e.g. on `np.sum`.